packages/code/src/SyntaxContext.tsx

```diff
@@ -5,6 +5,7 @@ interface SyntaxContext {
   highlightLines: LineHighlightingDefinition;
   showLineNumbers?: boolean;
   darkMode: boolean;
+  lineOffset?: number;
```

Comment:
I think this can be a required prop. In Code.tsx (line 150), you're setting a default for the prop, and since SyntaxContext is an internal component, we can write this to not ever be undefined.